src/git-duet/git-duet-rebase/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("rebase"))
```

Comment:
If `git rebase` doesn't support `--signoff`, what's the point of having the `git duet-rebase` subcommand?  Doesn't it do the same thing as `git rebase`?